Adds-missing-episdoes

kjaymiller opened the pull request:
Adds missing episodes in new format up to 28 Oct 2022

kjaymiller committed:
add content

kjaymiller committed:
add requirements

kjaymiller committed:
add automation files

kjaymiller committed:
run pre-commit

kjaymiller commented on site/content/2022-10-28.md:
```suggestion
podcast: 2b6aa41a
```

kjaymiller replied:
update podcast

kjaymiller committed:
Update site/content/2022-10-28.md

jonafato commented on site/CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1 @@
+CODE_OF_CONDUCT.md
```
This is a broken symlink. What was the intent here?

kjaymiller replied:
This can be removed... I'm not sure how to remove it honestly

jonafato commented on site/templates/page_template.md:
```diff
@@ -0,0 +1,16 @@
+---
```
This is a template used for the purpose of adding markdown content to the site, but it's not actually used by the site (i.e. during the compilation done by Render Engine) itself, correct? Can we move this to its own directory outside of `site`?

kjaymiller replied:
I originally had it at root but moved it to templates to clean the org up. There could be a directory call `publish` or something

jonafato replied:
Yeah, I'm good with something like that, just aiming to avoid content within `site` that's not actually required for the site build.

jonafato committed:
Fix yaml date parsing issue

YAML was automatically converting the `slug` attributes in the content
markdown files to dates, causing the site build to fail. This change
updates the existing slugs to quote them and the template to include
quotes for future generated files.

kjaymiller committed:
moves page to publish

kjaymiller committed:
removes archive

kjaymiller committed:
add 09-23

kjaymiller committed:
Merge branch 'adds-missing-episdoes' of github.com:Python-Community-News/Site into adds-missing-episdoes

kjaymiller committed:
Delete CODE_OF_CONDUCT.md
